src/main/kotlin/me/zeroeightsix/kami/Utils.kt

```diff
@@ -230,3 +229,15 @@ data class Colour(val a: Float, val r: Float, val g: Float, val b: Float) {
 }
 
 fun VertexConsumer.colour(colour: Colour) = this.color(colour.r, colour.g, colour.b, colour.a)
+
+fun<K, V> MutableMap<K, V>.put(p: Pair<K, V>) {
+    put(p.first, p.second)
+}
```

Comment:
Why is this needed?

Reply:
it makes it cleaner to add new entries to a map. it is used [here](https://github.com/LordMZTE/KAMI/blob/feature/baritonevariable/src/main/kotlin/me/zeroeightsix/kami/gui/text/VarMap.kt#L155)

Reply:
I didn't see the pair, my bad. I thought you were just doing a put of two values :p